wikihow2zim/scraper.py

```diff
@@ -625,20 +615,10 @@ def scrape_category_page(self, category: str, page_num: int, recurse: bool):
                     target_path=path,
                 )
 
-        return nb_pages, sub_categories
+        return nb_pages
 
     def scrape_article(self, article, remove_all_links=False):
-        if (
-            article in self.articles
-            or article in self.rewriter.exclusion_articles
-            or (
-                self.rewriter.inclusion_articles
-                and article not in self.rewriter.inclusion_articles
-            )
-        ):
-            return
         self.articles.add(article)
```

Comment:
Do we still need this? It's not only used to display stats at the end of the scrape. We already have a list of expected articles and missing articles. Only difference is the footer links… that are also in a list